src/Experimental/RAGTools/preparation.jl

```diff
@@ -227,6 +248,13 @@ function get_embeddings(
     return nothing
 end
 
+function get_embeddings(
```

Comment:
btw this is incredibly slow / inefficient for any practical purposes, thats why we need the batching